lib/create-file.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var File = require('vinyl'),
+	fs = require('fs');
+
+/**
+ * Creates a vinyl file
+ * @function createFile
+ *
+ * @param {STRING}  filePath  relative path to file
+ * @param {STRING}  cwd  current working directory
+ * @param {STRING}  base  directory where we start looking for the file
+ * @param {STRING}  type  stream|null
+ */
+exports.createFile = function createFile (filePath, cwd, base, type) {
+  var contents;
+
+  if (type == 'stream') {
+    contents = fs.createReadStream(filePath);
+  } else {
+    contents = fs.readFileSync(filePath);
```

Comment:
in case using fs.readFileSync i wanna suggest using this 
```javascript
try {
  data = fs.readFileSync(filePath);
} catch (e) {
  //ERROR handling
}